### DISCUSSION TITLE
Github google sso setup

### DISCUSSION THREAD

**Discussion by jchiang-genesis** [community/community#24619] [Enterprise]
<p>Hi All, i believe i followed the guide thoroughly to set up google as idp for github sso, but when i ran the test saml, it directed me to google sign as expected, but once i logged into the user acocunt, it says the <strong>Error: app_not_configured_for_user</strong> , I definitely allowed the app on google side for this user, i even tried enabled that saml app for all, can anyone enlighten me on this?</p>
<p>github sso setup guide on google</p><aside class="onebox allowlistedgeneric" data-onebox-src="https://support.google.com/a/answer/7553415?product_name=UnuFlow&amp;hl=en&amp;visit_id=637868439670256391-985190645&amp;rd=1&amp;src=supportwidget0&amp;hl=en#zippy=,step-set-up-google-as-a-saml-identity-provider-idp,step-enable-the-github-business-app,step-set-up-github-business-as-a-saml-service-provider-sp,step-verify-that-the-sso-is-working">
  <header class="source">

      <a href="https://support.google.com/a/answer/7553415?product_name=UnuFlow&amp;hl=en&amp;visit_id=637868439670256391-985190645&amp;rd=1&amp;src=supportwidget0&amp;hl=en#zippy=,step-set-up-google-as-a-saml-identity-provider-idp,step-enable-the-github-business-app,step-set-up-github-business-as-a-saml-service-provider-sp,step-verify-that-the-sso-is-working" target="_blank" rel="noopener nofollow ugc">support.google.com</a>
  </header>

  <article class="onebox-body">
    

<h3><a href="https://support.google.com/a/answer/7553415?product_name=UnuFlow&amp;hl=en&amp;visit_id=637868439670256391-985190645&amp;rd=1&amp;src=supportwidget0&amp;hl=en#zippy=,step-set-up-google-as-a-saml-identity-provider-idp,step-enable-the-github-business-app,step-set-up-github-business-as-a-saml-service-provider-sp,step-verify-that-the-sso-is-working" target="_blank" rel="noopener nofollow ugc">GitHub Business cloud application - Google Workspace Admin Help</a></h3>

  <p>You must be signed in as a super administrator for this task.
Using Security Assertion Markup Language (SAML), your users can use their Google Cloud credentials to sign in to enterprise-clo</p>

  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>

**Comment by jfullstackdev** [comment]
<p>just curious whether you are already setup as Super Admin,</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/d/6/d6a54218fb9e8d8da99c6058209ff1361d1af07c.png" data-download-href="/uploads/short-url/uCQf31JrqQ40gRcXXSKUJRkiY9e.png?dl=1" title="image"><img src="https://user-images.githubusercontent.com/47092464/181085685-3442442e-d2fa-4f03-933f-22a8752802a6.png" alt="image" data-base62-sha1="uCQf31JrqQ40gRcXXSKUJRkiY9e" width="690" height="165" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/d/6/d6a54218fb9e8d8da99c6058209ff1361d1af07c_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">741×178 15.2 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>
<p>you many want to check this out,</p>
<p><a href="https://cloud.google.com/resource-manager/docs/super-admin-best-practices#:~:text=A%20Google%20Workspace%20super%20admin,Google%20Cloud%2C%20and%20so%20forth" class="inline-onebox">Super administrator account best practices  |  Resource Manager Documentation  |  Google Cloud</a>.</p>

**Comment by jchiang-genesis** [comment, marked as answer]
<p>Hi, yea I did, actually I totally forgot to update everyone here, I just waited a day, somehow it just started working the next day, maybe the settings on google side or github side took some time to kick in. All is good now. thanks</p>
